pkg/compiler/codegen.go

```diff
@@ -194,20 +194,21 @@ func (c *codegen) emitStoreStructField(i int) {
 
 // getVarIndex returns variable type and position in corresponding slot,
 // according to current scope.
-func (c *codegen) getVarIndex(pkg string, name string) (varType, int) {
+func (c *codegen) getVarIndex(pkg string, name string) *varInfo {
```

Comment:
If it never returns `nil`, why does it return a pointer?

Reply:
Mostly to keep the interface same as with `varScope.getVarInfo`.